wordpress-mobile/WordPress-iOS
# Login design first pass

First pass at giving the login process the new visual design.

This is a hybrid design, so it looks a bit frankenstein. Effort was to get buttons, colors, fonts, looking right with approximately the correct layout. In the second pass, each screen (many newly created) will get its own PR for in-depth review.

![simulator screen shot may 26 2017 3 12 06 pm](https://cloud.githubusercontent.com/assets/517257/26512841/b184b88e-4225-11e7-859b-2933c6ee218a.png)
![simulator screen shot may 26 2017 3 10 48 pm](https://cloud.githubusercontent.com/assets/517257/26512842/b19e201c-4225-11e7-9add-8a607d0f0e73.png)

To test:
- make sure the login processes still work! Self-hosted, wp.com via user & pass, wp.com via magic link.

_Notes:_
- Signup hasn't been touched!

Needs review: @aerych 
_Welcome back!_
